lib/lib_rss.php

```diff
@@ -109,24 +109,29 @@ function safe_ascii($text) {
  * localhost address.
  *
  * @param $address the address to test, can be an IP or a URL.
- * @return true if server is accessible, false else.
+ * @return true if server is accessible, false otherwise.
  * @todo improve test with a more valid technique (e.g. test with an external server?)
  */
 function server_is_public($address) {
 	$host = parse_url($address, PHP_URL_HOST);
 
 	$is_public = !in_array($host, array(
-		'127.0.0.1',
 		'localhost',
 		'localhost.localdomain',
 		'[::1]',
+		'ip6-localhost',
 		'localhost6',
 		'localhost6.localdomain6',
 	));
 
-	return $is_public;
-}
+	if ($is_public) {
+		$ip = gethostbyname($host);
```

Comment:
This does not work as expected when FreshRSS is in a container (e.g. Træfik + Docker). Will fix.

Reply:
Fixed with https://github.com/FreshRSS/FreshRSS/pull/2084